src/widgets/ErrorDialog.cpp

```diff
@@ -158,6 +162,18 @@ void ShowErrorDialog(wxWindow *parent,
 }
 
 
+void ShowExceptionDialog(
+   wxWindow* parent, const TranslatableString& dlogTitle,
+   const TranslatableString& message, const wxString& helpPage, bool Close,
+   const wxString& log)
+{
+#ifndef HAS_SENTRY_REPORTING
+   ShowErrorDialog(parent, dlogTitle, message, helpPage, Close, log);
+#else
+   ShowErrorReportDialog(parent, dlogTitle, message, helpPage);
```

Comment:
Should you pass Close and log to ShowErrorReportDialog?

Reply:
`Close` defines the behavior of the help button. For Sentry dialog - there is no default option, so Close is always assumed to be `false`

Reply:
We always show the details always with the report. 

But I guess if `log` is not empty - it makes sense to include it with the report itself. I will add it